### PR TITLE
lint for 1.13.7

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import groovy.json.JsonSlurperClassic
 
-helpers = fileLoader.fromGit('helpers', 'https://github.com/keybase/jenkins-helpers.git', 'master', null, 'linux')
+helpers = fileLoader.fromGit('helpers', 'https://github.com/keybase/jenkins-helpers.git', 'master', null, 'linux-testing')
```
Comment (heronhaye) [inline Jenkinsfile]: will revert

File: Jenkinsfile [hunk 2]
```diff
@@ -36,7 +36,15 @@ def logKbwebServices(container) {
...
-helpers.rootLinuxNode(env, {
+def rootLinuxNode(env, handleError, cleanup, closure) {
+  if (env.CHANGE_TITLE && env.CHANGE_TITLE.contains('[ci-skip]')) {
+    println "Skipping build because PR title contains [ci-skip]"
+  } else {
+    helpers.nodeWithDockerCleanup("linux-testing", handleError, cleanup, closure)
+  }
+}
+
+rootLinuxNode(env, {
```
Comment (heronhaye) [inline Jenkinsfile]: will revert

File: go/client/chat_cli_ui.go [hunk 3]
```diff
@@ -253,7 +253,7 @@ func (c *ChatCLIUI) ChatSearchInboxHit(ctx context.Context, arg chat1.ChatSearch
 	if width > 80 {
 		width = 80
 	}
-	fmt.Fprintf(w, fmt.Sprintf("%s\n", strings.Repeat("-", width)))
+	fmt.Fprint(w, fmt.Sprintf("%s\n", strings.Repeat("-", width)))
```
Comment (joshblum) [inline go/client/chat_cli_ui.go]: drop the sprintf instead?

File: go/client/saltpack_ui.go [hunk 4]
```diff
@@ -82,21 +82,21 @@ func (s *SaltpackUI) SaltpackPromptForDecrypt(_ context.Context, arg keybase1.Sa
 	w := s.terminal.ErrorWriter()
 	switch arg.Sender.SenderType {
 	case keybase1.SaltpackSenderType_TRACKING_OK:
-		fmt.Fprintf(w, ColorString(s.G(), "green", fmt.Sprintf("Authored by %s.\n", ColorString(s.G(), "bold", arg.Sender.Username))))
```
Comment (joshblum) [inline go/client/saltpack_ui.go]: why not drop the inner sprintf throughout?

File: go/Makefile [hunk 5]
```diff
@@ -89,4 +89,12 @@ shavendorstellar:
 	echo "expecting f3a7d9906dab5c481669b5b4f852bf447388bd3d:"
 	find vendor/github.com/stellar -type f -print0 | sort -z | xargs -0 shasum | shasum
 
+GOLANGCI_RUN_OPT = --color auto
```
Comment (joshblum) [inline go/Makefile]: are these commands needed?

**Pull request merged**